### PR TITLE
Disable octal literals rubocop

### PR DESCRIPTION
**Pull request by armiller**

### CODE REVIEW

File: files/default/rubocop.yml [hunk 1]
```diff
@@ -4,3 +4,6 @@ AllCops:
     - '**/Cheffile'
   Exclude:
     - 'metadata.rb'
+
+Style/NumericLiteralPrefix:
+  Enabled: false
```
Comment (ramereth) [inline files/default/rubocop.yml]: Would using `EnforcedOctalStyle` => `zero_only` be a better option for this instead? See [upstream docs](http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/NumericLiteralPrefix).

**Comment by ramereth** [comment]
:+1:

**Comment by armiller** [comment]
!bump patch

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.0.4.  Have a nice day!

**Pull request merged**